README.md

```diff
@@ -1,4 +1,67 @@
-# **Brain 4CE** - An EEG-based Brain-Computer Interface
-> *Where thoughts become motion*.
-## Designed by BU ECE 2023 Senior Design Capstone Team 4: 
-**Alex Johnson, Dayanna De La Torres, Jonathan Mikalov, Brendan Shortall, & Mitchell Gilmore**
\ No newline at end of file
+<div style="text-align: center;">
+
+# **Brain-4ce** - An EEG-based Brain-Computer Interface
+
+</div>
+
+
+<p align="center">
+  <img src="./Documents/brain_4ce_logo.png" alt="Brain 4ce Logo" style="display:block; margin:auto;" width="350">
+  <br>
+  <em>Where thoughts become motion.</em>
+</p>
+<!-- <br></br> -->
+<div style="text-align: center;">
+  <h2 style="text-decoration: underline;">Designed by BU ECE 2023 Senior Design Capstone Team 4</h2>
+  <img src="./Documents/team_photo.png" alt="Brain 4ce Logo" style="display:block; margin:auto;" width="600">
```

Comment:
maybe center this photo as well and make it same width?

Reply:
Keeping the photo as is looks best in my opinion. The centering should be fixed once I push this next commit.